sympy/tensor/indexed.py

```diff
@@ -1,115 +1,10 @@
-"""Module that defines indexed objects
```

Comment:
Why did you move this to the docs instead of having it auto-import into the docs?